plugins/eslint-aws/lib/rules/no-mocha-arrows.ts

```diff
@@ -0,0 +1,59 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as eslint from 'eslint'
+
+const MOCHA_EXPRESSIONS = new Set(['after', 'before', 'it', 'describe', 'afterEach', 'beforeEach'])
+
+type ArrowNode = eslint.Rule.Node & { type: 'ArrowFunctionExpression' }
+
+function getArrow(node: eslint.Rule.Node): ArrowNode | undefined {
+    if (node.type !== 'CallExpression') {
+        return undefined
+    }
+
+    if (node.callee.type === 'Identifier' && MOCHA_EXPRESSIONS.has(node.callee.name)) {
+        return <any>node.arguments.find(arg => arg.type === 'ArrowFunctionExpression')
+    }
+}
+
+// https://eslint.org/docs/developer-guide/working-with-rules
```

Comment:
url was redirected:

```suggestion
// https://eslint.org/docs/latest/extend/custom-rules
```

surprised that `ESTree` is not mentioned in this PR diff, I guess there's no typescript definitions for it?